packages/babel-parser/src/parser/comments.js

```diff
@@ -264,7 +264,17 @@ export default class CommentsParser extends BaseParser {
       ) {
         node.innerComments = trailingComments;
       } else {
-        node.trailingComments = trailingComments;
+        // TrailingComments maybe contain innerComments
+        const findIndex = trailingComments.findIndex(
```

Comment:
Can we name this variable `firstTrailingCommentIndex`? We could probably get ride of the comment then!